src/Cms/ModelWithContent.php

```diff
@@ -443,7 +445,10 @@ public function save(
 		$clone = $this->clone();
 
 		// move the old model into memory
-		$this->changeStorage(ImmutableMemoryStorage::class);
+		$this->changeStorage(
+			toStorage: ImmutableMemoryStorage::class,
+			copy: true
+		);
```

Comment:
```suggestion
		$this->changeStorage(ImmutableMemoryStorage::class, copy: true);
```

Reply:
I would ignore this one. It's going to change in the next PR anyway when we need to pass the clone. :)